se-test/README.md

```diff
@@ -1,18 +1,28 @@
 # Overview
 This task is intended for people who interview for our Software Engineer roles.
 
-Please make sure you read [our expectations](../README.md#what-we-are-looking-for) first before starting the following tasks:  
+Please make sure you read [our expectations](../README.md#what-we-are-looking-for) first before starting on any of the below options:   
 
 ## Option 1
 
-Build a JavaScript application that shows the latest news from the United Kingdom, using the [newsapi.org](https://newsapi.org) service.
+Build a JavaScript application that shows the latest news world-wide, using the [newsapi.org](https://newsapi.org) service.
 
-Running the app we should be able to see the latest **UK news**. The user should have the ability to filter by keywords (see [everything endpoint api](https://newsapi.org/docs/endpoints/everything)). Optionally, if the user clicks on a news item it will navigate to a page displaying the full news article selected.
+Running the app we should be able to see the latest **news**. The user should have the ability to filter by keywords (see [everything endpoint api](https://newsapi.org/docs/endpoints/everything)). Optionally, if the user clicks on a news item it will navigate to a page displaying the full news article selected.
 
 This is a full stack challenge and we are expecting both a Frontend application and a Backend application that communicate with each other.
 
 A Full Stack developer is someone with familiarity in each layer; you might be stronger on the frontend, or vice-versa. Feel free to focus more on what you know best, be creative and show us your skills!
 
+## Option 2
+
+If you have some existing code that you would be prepared to share with us, we can look at that instead. The code should
+meet the following criteria:
+
+* It should be similar hours of your own work as with Option 1 (as opposed to others' or your team's work for example)
+* Ideally, it should be a web/mobile app that involves both client and server-side skills
+* It should demonstrate how you approach a problem with a simple solution that you're confident to walk us through in an interview scene (as opposed to sharing an app of 10k LOC for example) 
+* It should be something that's legit to share with us (as opposed to NDA protected stuff for example)
```

Comment:
How about: 'It should be something you are permitted to share with us (not confidential or protected by NDA for example)' ?